pkgs/servers/http/jboss/jdbc/mysql/default.nix

```diff
@@ -2,11 +2,18 @@
 
 stdenv.mkDerivation {
   pname = "jboss-mysql-jdbc";
+  version = mysql_jdbc.version;
 
-  builder = ./builder.sh;
+  dontUnpack = true;
 
-  inherit mysql_jdbc;
-  version = mysql_jdbc.version;
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/server/default/lib
+    ln -s $mysql_jdbc/share/java/mysql-connector-java.jar $out/server/default/lib/mysql-connector-java.jar
+
+    runHook postInstall
+  '';
 
   meta = {
     platforms = lib.platforms.unix;
```

Comment:
It would be nice to inherit some more meta.

Reply:
should it inherit homepage?
should a string be appended before the description?

Reply:
I don't know for sure. I just saw it a little empty.